Sdks/AdminApi.Sdk/src/Endpoints/Clients/Types/ClientInfo.cs

```diff
@@ -6,6 +6,5 @@ public class ClientInfo
     public required string DisplayName { get; set; }
     public required string DefaultTier { get; set; }
     public required DateTime CreatedAt { get; set; }
-    public required int NumberOfIdentities { get; set; }
```

Comment:
I don't get it. Why did you remove `NumberOfIdentities` now?

Reply:
By mistake. I did not notice this while manually testing since it only appears when SDK is used, and I didn't check responses in the tests.

Reply:
That's why I keep saying "review your own changes before you commit them" (this is just a reminder - of course you miss something from time to time)

Reply:
I did review it. I got the response as expected when I manually tested with a client (Insomnia). But the problem was that the `ClientInfo` is only used by the SDK which is (at the moment) only used by the integration tests. That is why it slipped by, it wasn't checked by anything if it were there, and so... [continued](https://github.com/nmshd/backbone/pull/797#discussion_r1715220005)

Reply:
> I got the response as expected when I manually tested with a client (Insomnia)

I'm not talking about manual testing, I'm talking about looking at the code changes before you `git stage` them. Such a review usually uncovers a problem like accidentally deleting something.